service/src/test/java/com/ge/predix/acs/testutils/XFiles.java

```diff
@@ -85,17 +87,23 @@ private XFiles() {
 
         BaseSubject specialAgentsGroup = new BaseSubject(SPECIAL_AGENTS_GROUP);
         specialAgentsGroup.setAttributes(SPECIAL_AGENTS_GROUP_ATTRIBUTES);
-        specialAgentsGroup
+
+        if (titanProfileActive()) {
+            specialAgentsGroup
                 .setParents(new HashSet<>(Arrays.asList(new Parent[] { new Parent(fbi.getSubjectIdentifier()) })));
+        }
 
         BaseSubject topSecretGroup = new BaseSubject(TOP_SECRET_GROUP);
         topSecretGroup.setAttributes(TOP_SECRET_GROUP_ATTRIBUTES);
 
         BaseSubject agentMulder = new BaseSubject(AGENT_MULDER);
         agentMulder.setAttributes(MULDERS_ATTRIBUTES);
-        agentMulder.setParents(
+
+        if (titanProfileActive()) {
```

Comment:
per our discussion, this class should only be used for tests specific to hierarchical attributes.  Lets fix the test which is using this with postgres profile instead. And remove the profile checks from this class.